contracts/coordinator/src/msg.rs

```diff
@@ -16,15 +15,15 @@ pub enum ExecuteMsg {
     #[permission(Governance)]
     RegisterProverContract {
         chain_name: ChainName,
-        new_prover_addr: Addr,
+        new_prover_addr: String,
     },
     #[permission(Specific(prover))]
-    SetActiveVerifiers { verifiers: HashSet<Addr> },
+    SetActiveVerifiers { verifiers: HashSet<String> },
```

Comment:
We should introduce a `VerifierAddress` type to length restrict verifier addresses and expose a validation function
https://axelarnetwork.atlassian.net/browse/AXE-4586

Reply:
yeah, i will do this in a follow up